eclass/netsurf.eclass

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: netsurf.eclass
+# @MAINTAINER:
+# maintainer-needed@gentoo.org
+# @SUPPORTED_EAPIS: 5 6 7
```

Comment:
It looks like all of the ebuilds using the eclass are on EAPI=7? If so, that would be great:

  * In EAPI=5, get_libdir isn't available without inheriting another eclass
  * Before EAPI=7, variables like EPREFIX don't have a trailing slash (so you need more code to add it conditionally)

If this can be changed to support only EAPI=7, you should add something like the following (stolen from xorg-3.eclass) to enforce it:

```sh
case "${EAPI:-0}" in
	7) ;;
	*) die "EAPI=${EAPI} is not supported" ;;
esac
```